lib/wasi/src/lib.rs

```diff
@@ -172,7 +82,7 @@ impl WasiEnv {
 
     /// Get a reference to the memory
     pub fn memory(&self) -> &Memory {
-        self.memory.get_memory().expect("The expected Memory is not attached to the `WasiEnv`. Did you forgot to call wasi_env.set_memory(...)?")
+        self.memory_ref().unwrap()
```

Comment:
Use expect rather than unwrap.

Reply:
The idea was that It shouldn't be possible to use this incorrectly anymore, but if done manually then I suppose it could be used incorrectly